selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/NavigateToFile.java

```diff
@@ -159,7 +159,7 @@ public void selectFileByName(String nameOfFile) {
             .until(
                 visibilityOfElementLocated(
                     By.xpath(format(Locators.FILE_NAME_LIST_SELECT, nameOfFile))));
-
+    webElement.click();
     WaitUtils.sleepQuietly(500, MILLISECONDS);
```

Comment:
Suggest to consider replacing **WaitUtils.sleepQuietly** on **TestWebElementRenderChecker.waitElementIsRendered**.